plenum/test/view_change/test_view_change_by_catchup_and_order.py

```diff
@@ -20,7 +20,7 @@ def tconf(tconf):
             yield tconf
 
 
-@pytest.mark.skip(reason="INDY-2223: Temporary skipped to create build")
+@pytest.mark.skip(reason="All commits before vc will be cleared and we don't order during vc")
```

Comment:
I think this test still makes sense ans should pass with the current code. Commits coming during view change will be discarded, but we will do re-ordering in new view, and all batches we have prepared certificates for will be ordered eventually